server/README.md

```diff
@@ -50,8 +50,13 @@ curl -i \
 	-X POST \
 	-d '{isoCountryCode: CH}' \
 	'https://whoapp.org/WhoService/putLocation'
+```
+
+#### Static Content
 
-# Static Content - served from Google Cloud Storage
+Served from Google Cloud Storage:
+
+```
 curl https://storage.googleapis.com/who-myhealth-staging-static-content-01/\
```

Comment:
and the curl line below will change as well, but not yet.

Reply:
agreed